### PR TITLE
Cool.gyms

### PR DESCRIPTION
**Pull request by arrants8**

### CODE REVIEW

File: free_court/src/components/Gym/styles.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .name {
-	text-align: center;
+	/*margin-left: 50%;*/
```
Comment (Mark-Jung) [inline free_court/src/components/Gym/styles.css]: If you are not using it, get rid of it.

File: free_court/src/ducks/main.js [hunk 2]
```diff
@@ -64,6 +69,7 @@ export const load_main_success = (dispatch, response) => {
 }
 
 export const load_main_failure = (dispatch, error) => {
+    console.log("hi");
```
Comment (Mark-Jung) [inline free_court/src/ducks/main.js]: delete all debugging console logs

**Comment by jdk27** [comment]
Includes title, gym etc in mapStateToProps. Needs to be removed when connected to db

File: free_court/src/components/Gym/Gym.js [hunk 3]
```diff
@@ -1,15 +1,44 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
```
Comment (Mark-Jung) [inline free_court/src/components/Gym/Gym.js]: If you are not using some library, delete it.

**Review by Mark-Jung — CHANGES_REQUESTED** [review]
Small changes.

**Review by Mark-Jung — APPROVED** [review]

**Pull request merged**